### ISSUE TITLE
Istio 1.8 version istio-policy charts missing error in istio-discovery folder path

### ISSUE THREAD

**Issue by shahriak** [istio/istio#31541]
I am trying to deploy istio version 1.8.4 using istio-operator and helm and running into below issue. I was able to deploy istio version 1.7.7 without any issues previously, the only change for the 1.8.4 version were removing the istio-policy and istio-telemetry charts and some other minor changes to istioOperator file.
```

2021-03-18T03:35:42.065478Z     info    ControlZ available at 127.0.0.1:9876
2021-03-18T03:35:42.065604Z     info    leader election cm: istio-operator-lock
2021-03-18T03:35:42.468944Z     info    Registering Components.
2021-03-18T03:35:42.469014Z     info    Adding controller for IstioOperator
2021-03-18T03:35:42.469111Z     info    Controller added
2021-03-18T03:35:42.469120Z     info    Starting the Cmd.
2021-03-18T03:35:42.469284Z     info    attempting to acquire leader lease  istio-operator/istio-operator-lock...
2021-03-18T03:35:42.483347Z     info    successfully acquired lease istio-operator/istio-operator-lock
2021-03-18T03:35:44.691987Z     info    Reconciling IstioOperator
2021-03-18T03:35:44.693363Z     info    Adding finalizer istio-finalizer.install.istio.io to istio-system/stiocontrolplane
2021-03-18T03:35:44.705984Z     info    Updating IstioOperator
2021-03-18T03:35:44.709167Z     info    proto: tag has too few fields: "-"
2021-03-18T03:35:44.870529Z     info    Detecting third-party JWT support
2021-03-18T03:35:45.292971Z     info    found symbolic link in path: /istio/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml resolves to /istio/manifests/charts/istio-control/istio-discovery/templates/..2021_03_18_03_35_06.761447241/telemetryv2_1.8.yaml
2021-03-18T03:35:45.293036Z     info    found symbolic link in path: /istio/manifests/charts/istio-control/istio-discovery/values.yaml resolves to /istio/manifests/charts/istio-control/istio-discovery/..2021_03_18_03_35_05.009717083/values.yaml
profile: "default"
install_package_path: "/istio/manifests"
hub: "acr.azurecr.io/istio"
tag: 1.7 .7 - 2021030501 mesh_config: < key: "defaultConfig"
value: map[proxyMetadata: map[] terminationDrainDuration: 30 s] > mesh_config: < key: "enablePrometheusMerge"
value: true > components: < base: < enabled: < BoolValue: < value: true > > > pilot: < enabled: < BoolValue: < value: true > > tag: < nil > spec: < nil > k8s: < env: < name: "PILOT_SKIP_VALIDATE_TRUST_DOMAIN"
value: "true" > hpa_spec: < minReplicas: 2 > readiness_probe: < httpGet: < path: "/ready"
port: < IntOrString: < type: 0 intVal: 8080 strVal: "" > > > initialDelaySeconds: 1 timeoutSeconds: 5 periodSeconds: 3 > strategy: < rollingUpdate: < maxUnavailable: < IntOrString: < type: 1 intVal: 0 strVal: "25%" > > maxSurge: < IntOrString: < type: 1 intVal: 0 strVal: "100%" > > > > overlays: < kind: "Deployment"
name: "istiod"
patches: < path: "spec.template.spec.dnsConfig"
value: map[options: [map[name: single - request - reopen]]] > > > > cni: < enabled: < BoolValue: < > > tag: < nil > spec: < nil > > istiod_remote: < enabled: < BoolValue: < > > tag: < nil > spec: < nil > > ingress_gateways: < enabled: < BoolValue: < value: true > > name: "istio-ingressgateway"
tag: < nil > k8s: < env: < name: "ISTIO_META_ROUTER_MODE"
value: "sni-dnat" > env: < name: "ISTIO_META_IDLE_TIMEOUT"
value: "30s" > hpa_spec: < scaleTargetRef: < kind: "Deployment"
name: "istio-ingressgateway"
apiVersion: "apps/v1" > minReplicas: 2 maxReplicas: 5 metrics: < type: "Resource"
resource: < name: "cpu"
targetAverageUtilization: 60 > > > node_selector: < key: "kubernetes.io/os"
value: "linux" > resources: < limits: < key: "cpu"
value: "2000m" > limits: < key: "memory"
value: "1024Mi" > requests: < key: "cpu"
value: "100m" > requests: < key: "memory"
value: "128Mi" > > service: < ports: < name: "https"
port: 443 targetPort: < IntOrString: < type: 0 intVal: 32443 strVal: "" > > > ports: < name: "https-istio"
port: 16443 targetPort: < IntOrString: < type: 0 intVal: 16443 strVal: "" > > > ports: < name: "https-mutual"
port: 17443 targetPort: < IntOrString: < type: 0 intVal: 17443 strVal: "" > > > > strategy: < rollingUpdate: < maxUnavailable: < IntOrString: < type: 1 intVal: 0 strVal: "25%" > > maxSurge: < IntOrString: < type: 1 intVal: 0 strVal: "100%" > > > > overlays: < kind: "Deployment"
name: "istio-ingressgateway"
patches: < path: "spec.template.spec.dnsConfig"
value: map[options: [map[name: single - request - reopen]]] > > > > egress_gateways: < enabled: < BoolValue: < > > name: "istio-egressgateway"
tag: < nil > k8s: < env: < name: "ISTIO_META_ROUTER_MODE"
value: "standard" > hpa_spec: < scaleTargetRef: < kind: "Deployment"
name: "istio-egressgateway"
apiVersion: "apps/v1" > minReplicas: 1 maxReplicas: 5 metrics: < type: "Resource"
resource: < name: "cpu"
targetAverageUtilization: 80 > > > resources: < limits: < key: "cpu"
value: "2000m" > limits: < key: "memory"
value: "1024Mi" > requests: < key: "cpu"
value: "100m" > requests: < key: "memory"
value: "128Mi" > > service: < ports: < name: "http2"
protocol: "TCP"
port: 80 targetPort: < IntOrString: < type: 0 intVal: 8080 strVal: "" > > > ports: < name: "https"
protocol: "TCP"
port: 443 targetPort: < IntOrString: < type: 0 intVal: 8443 strVal: "" > > > ports: < name: "tls"
protocol: "TCP"
port: 15443 targetPort: < IntOrString: < type: 0 intVal: 15443 strVal: "" > > > > strategy: < rollingUpdate: < maxUnavailable: < IntOrString: < type: 1 intVal: 0 strVal: "25%" > > maxSurge: < IntOrString: < type: 1 intVal: 0 strVal: "100%" > > > > > > > addon_components: < key: "istiocoredns"
value: < enabled: < BoolValue: < > > spec: < nil > > > values: < key: "base"
value: map[enableCRDTemplates: false validationURL: ] > values: < key: "clusterResources"
value: true > values: < key: "cni"
value: map[excludeNamespaces: [istio - system kube - system] logLevel: info] > values: < key: "gateways"
value: map[istio - egressgateway: map[autoscaleEnabled: true env: map[] name: istio - egressgateway secretVolumes: [map[mountPath: /etc/istio / egressgateway - certs name: egressgateway - certs secretName: istio - egressgateway - certs] map[mountPath: /etc/istio / egressgateway - ca - certs name: egressgateway - ca - certs secretName: istio - egressgateway - ca - certs]] type: ClusterIP zvpn: map[]] istio - ingressgateway: map[autoscaleEnabled: true autoscaleMin: 2 env: map[] externalTrafficPolicy: Local name: istio - ingressgateway podAntiAffinityTermLabelSelector: [map[key: app operator: In topologyKey: kubernetes.io / os values: istio - ingressgateway]] ports: [map[name: https port: 443 targetPort: 32443]] secretVolumes: [map[mountPath: /etc/istio / ingressgateway - certs name: ingressgateway - certs secretName: istio - ingressgateway - certs] map[mountPath: /etc/istio / ingressgateway - ca - certs name: ingressgateway - ca - certs secretName: istio - ingressgateway - ca - certs]] type: LoadBalancer zvpn: map[]]] > values: < key: "global"
value: map[arch: map[amd64: 2 ppc64le: 2 s390x: 2] configValidation: true defaultNodeSelector: map[kubernetes.io / os: linux] defaultPodDisruptionBudget: map[enabled: true] defaultResources: map[requests: map[cpu: 10 m]] imagePullPolicy: imagePullSecrets: [] istioNamespace: istio - system istiod: map[enableAnalysis: false] jwtPolicy: third - party - jwt logAsJson: false logging: map[level: default: info] meshExpansion: map[enabled: false useILB: false] meshNetworks: map[] mountMtlsCerts: false multiCluster: map[clusterName: aks - meshtest - zb06 - uswest2 enabled: true globalDomainSuffix: global includeEnvoyFilter: true] network: omitSidecarInjectorConfigMap: false oneNamespace: false operatorManageWebhooks: false pilotCertProvider: istiod priorityClassName: proxy: map[autoInject: enabled clusterDomain: cluster.local componentLogLevel: misc: error enableCoreDump: false excludeIPRanges: excludeInboundPorts: excludeOutboundPorts: holdApplicationUntilProxyStarts: true image: proxyv2 includeIPRanges: * logLevel: warning privileged: false readinessFailureThreshold: 30 readinessInitialDelaySeconds: 1 readinessPeriodSeconds: 2 resources: map[limits: map[cpu: 2000 m memory: 1024 Mi] requests: map[cpu: 100 m memory: 128 Mi]] statusPort: 15020 tracer: zipkin] proxy_init: map[image: proxyv2 resources: map[limits: map[cpu: 2000 m memory: 1024 Mi] requests: map[cpu: 10 m memory: 10 Mi]]] sds: map[token: map[aud: istio - ca]] sts: map[servicePort: 0] tracer: map[datadog: map[] lightstep: map[] stackdriver: map[] zipkin: map[]] trustDomain: aks - meshtest - zb06 - uswest2 useMCP: false] > values: < key: "istiocoredns"
value: map[coreDNSImage: coredns / coredns coreDNSPluginImage: istio / coredns - plugin: 0.2 - istio - 1.1 coreDNSTag: 1.6 .2] > values: < key: "istiodRemote"
value: map[injectionURL: ] > values: < key: "pilot"
value: map[autoscaleEnabled: true autoscaleMax: 5 autoscaleMin: 1 configMap: true cpu: map[targetAverageUtilization: 80] deploymentLabels: < nil > enableProtocolSniffingForInbound: true enableProtocolSniffingForOutbound: true env: map[PILOT_SKIP_VALIDATE_TRUST_DOMAIN: true] image: pilot keepaliveMaxServerConnectionAge: 30 m nodeSelector: map[] replicaCount: 1 traceSampling: 1] > values: < key: "sidecarInjectorWebhook"
value: map[enableNamespacesByDefault: false objectSelector: map[autoInject: true enabled: false] rewriteAppHTTPProbe: true] > values: < key: "telemetry"
value: map[enabled: true v2: map[enabled: false metadataExchange: map[wasmEnabled: false] prometheus: map[enabled: true wasmEnabled: false] stackdriver: map[configOverride: map[] enabled: false logging: false monitoring: false topology: false]]] >
	error: stat / istio / manifests / charts / istio - policy: no such file or directory
profile: "default"
install_package_path: "/istio/manifests"
hub: "acr.azurecr.io/istio"
tag: 1.7 .7 - 2021030501 mesh_config: < key: "defaultConfig"
value: map[proxyMetadata: map[] terminationDrainDuration: 30 s] > mesh_config: < key: "enablePrometheusMerge"
value: true > components: < base: < enabled: < BoolValue: < value: true > > > pilot: < enabled: < BoolValue: < value: true > > tag: < nil > spec: < nil > k8s: < env: < name: "PILOT_SKIP_VALIDATE_TRUST_DOMAIN"
value: "true" > hpa_spec: < minReplicas: 2 > readiness_probe: < httpGet: < path: "/ready"
port: < IntOrString: < type: 0 intVal: 8080 strVal: "" > > > initialDelaySeconds: 1 timeoutSeconds: 5 periodSeconds: 3 > strategy: < rollingUpdate: < maxUnavailable: < IntOrString: < type: 1 intVal: 0 strVal: "25%" > > maxSurge: < IntOrString: < type: 1 intVal: 0 strVal: "100%" > > > > overlays: < kind: "Deployment"
name: "istiod"
patches: < path: "spec.template.spec.dnsConfig"
value: map[options: [map[name: single - request - reopen]]] > > > > cni: < enabled: < BoolValue: < > > tag: < nil > spec: < nil > > istiod_remote: < enabled: < BoolValue: < > > tag: < nil > spec: < nil > > ingress_gateways: < enabled: < BoolValue: < value: true > > name: "istio-ingressgateway"
tag: < nil > k8s: < env: < name: "ISTIO_META_ROUTER_MODE"
value: "sni-dnat" > env: < name: "ISTIO_META_IDLE_TIMEOUT"
value: "30s" > hpa_spec: < scaleTargetRef: < kind: "Deployment"
name: "istio-ingressgateway"
apiVersion: "apps/v1" > minReplicas: 2 maxReplicas: 5 metrics: < type: "Resource"
resource: < name: "cpu"
targetAverageUtilization: 60 > > > node_selector: < key: "kubernetes.io/os"
value: "linux" > resources: < limits: < key: "cpu"
value: "2000m" > limits: < key: "memory"
value: "1024Mi" > requests: < key: "cpu"
value: "100m" > requests: < key: "memory"
value: "128Mi" > > service: < ports: < name: "https"
port: 443 targetPort: < IntOrString: < type: 0 intVal: 32443 strVal: "" > > > ports: < name: "https-istio"
port: 16443 targetPort: < IntOrString: < type: 0 intVal: 16443 strVal: "" > > > ports: < name: "https-mutual"
port: 17443 targetPort: < IntOrString: < type: 0 intVal: 17443 strVal: "" > > > > strategy: < rollingUpdate: < maxUnavailable: < IntOrString: < type: 1 intVal: 0 strVal: "25%" > > maxSurge: < IntOrString: < type: 1 intVal: 0 strVal: "100%" > > > > overlays: < kind: "Deployment"
name: "istio-ingressgateway"
patches: < path: "spec.template.spec.dnsConfig"
value: map[options: [map[name: single - request - reopen]]] > > > > egress_gateways: < enabled: < BoolValue: < > > name: "istio-egressgateway"
tag: < nil > k8s: < env: < name: "ISTIO_META_ROUTER_MODE"
value: "standard" > hpa_spec: < scaleTargetRef: < kind: "Deployment"
name: "istio-egressgateway"
apiVersion: "apps/v1" > minReplicas: 1 maxReplicas: 5 metrics: < type: "Resource"
resource: < name: "cpu"
targetAverageUtilization: 80 > > > resources: < limits: < key: "cpu"
value: "2000m" > limits: < key: "memory"
value: "1024Mi" > requests: < key: "cpu"
value: "100m" > requests: < key: "memory"
value: "128Mi" > > service: < ports: < name: "http2"
protocol: "TCP"
port: 80 targetPort: < IntOrString: < type: 0 intVal: 8080 strVal: "" > > > ports: < name: "https"
protocol: "TCP"
port: 443 targetPort: < IntOrString: < type: 0 intVal: 8443 strVal: "" > > > ports: < name: "tls"
protocol: "TCP"
port: 15443 targetPort: < IntOrString: < type: 0 intVal: 15443 strVal: "" > > > > strategy: < rollingUpdate: < maxUnavailable: < IntOrString: < type: 1 intVal: 0 strVal: "25%" > > maxSurge: < IntOrString: < type: 1 intVal: 0 strVal: "100%" > > > > > > > addon_components: < key: "istiocoredns"
value: < enabled: < BoolValue: < > > spec: < nil > > > values: < key: "base"
value: map[enableCRDTemplates: false validationURL: ] > values: < key: "clusterResources"
value: true > values: < key: "cni"
value: map[excludeNamespaces: [istio - system kube - system] logLevel: info] > values: < key: "gateways"
value: map[istio - egressgateway: map[autoscaleEnabled: true env: map[] name: istio - egressgateway secretVolumes: [map[mountPath: /etc/istio / egressgateway - certs name: egressgateway - certs secretName: istio - egressgateway - certs] map[mountPath: /etc/istio / egressgateway - ca - certs name: egressgateway - ca - certs secretName: istio - egressgateway - ca - certs]] type: ClusterIP zvpn: map[]] istio - ingressgateway: map[autoscaleEnabled: true autoscaleMin: 2 env: map[] externalTrafficPolicy: Local name: istio - ingressgateway podAntiAffinityTermLabelSelector: [map[key: app operator: In topologyKey: kubernetes.io / os values: istio - ingressgateway]] ports: [map[name: https port: 443 targetPort: 32443]] secretVolumes: [map[mountPath: /etc/istio / ingressgateway - certs name: ingressgateway - certs secretName: istio - ingressgateway - certs] map[mountPath: /etc/istio / ingressgateway - ca - certs name: ingressgateway - ca - certs secretName: istio - ingressgateway - ca - certs]] type: LoadBalancer zvpn: map[]]] > values: < key: "global"
value: map[arch: map[amd64: 2 ppc64le: 2 s390x: 2] configValidation: true defaultNodeSelector: map[kubernetes.io / os: linux] defaultPodDisruptionBudget: map[enabled: true] defaultResources: map[requests: map[cpu: 10 m]] imagePullPolicy: imagePullSecrets: [] istioNamespace: istio - system istiod: map[enableAnalysis: false] jwtPolicy: third - party - jwt logAsJson: false logging: map[level: default: info] meshExpansion: map[enabled: false useILB: false] meshNetworks: map[] mountMtlsCerts: false multiCluster: map[clusterName: aks - meshtest - zb06 - uswest2 enabled: true globalDomainSuffix: global includeEnvoyFilter: true] network: omitSidecarInjectorConfigMap: false oneNamespace: false operatorManageWebhooks: false pilotCertProvider: istiod priorityClassName: proxy: map[autoInject: enabled clusterDomain: cluster.local componentLogLevel: misc: error enableCoreDump: false excludeIPRanges: excludeInboundPorts: excludeOutboundPorts: holdApplicationUntilProxyStarts: true image: proxyv2 includeIPRanges: * logLevel: warning privileged: false readinessFailureThreshold: 30 readinessInitialDelaySeconds: 1 readinessPeriodSeconds: 2 resources: map[limits: map[cpu: 2000 m memory: 1024 Mi] requests: map[cpu: 100 m memory: 128 Mi]] statusPort: 15020 tracer: zipkin] proxy_init: map[image: proxyv2 resources: map[limits: map[cpu: 2000 m memory: 1024 Mi] requests: map[cpu: 10 m memory: 10 Mi]]] sds: map[token: map[aud: istio - ca]] sts: map[servicePort: 0] tracer: map[datadog: map[] lightstep: map[] stackdriver: map[] zipkin: map[]] trustDomain: aks - meshtest - zb06 - uswest2 useMCP: false] > values: < key: "istiocoredns"
value: map[coreDNSImage: coredns / coredns coreDNSPluginImage: istio / coredns - plugin: 0.2 - istio - 1.1 coreDNSTag: 1.6 .2] > values: < key: "istiodRemote"
value: map[injectionURL: ] > values: < key: "pilot"
value: map[autoscaleEnabled: true autoscaleMax: 5 autoscaleMin: 1 configMap: true cpu: map[targetAverageUtilization: 80] deploymentLabels: < nil > enableProtocolSniffingForInbound: true enableProtocolSniffingForOutbound: true env: map[PILOT_SKIP_VALIDATE_TRUST_DOMAIN: true] image: pilot keepaliveMaxServerConnectionAge: 30 m nodeSelector: map[] replicaCount: 1 traceSampling: 1] > values: < key: "sidecarInjectorWebhook"
value: map[enableNamespacesByDefault: false objectSelector: map[autoInject: true enabled: false] rewriteAppHTTPProbe: true] > values: < key: "telemetry"
value: map[enabled: true v2: map[enabled: false metadataExchange: map[wasmEnabled: false] prometheus: map[enabled: true wasmEnabled: false] stackdriver: map[configOverride: map[] enabled: false logging: false monitoring: false topology: false]]] >
	error: stat / istio / manifests / charts / istio - policy: no such file or directory
```

**Bug description**
The istio-policy and istio-telemetry charts have been removed as of 1.8, I am not sure why it's complaining it's not able to find the policy charts

[ x] Docs
[ x] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure
[ ] Upgrade

**Expected behavior**
Istio should install successfully without any errors

**Steps to reproduce the bug**
Install istio using helm and istioOperator

**Version** (include the output of `istioctl version --remote` and `kubectl version --short` and `helm version --short` if you used Helm)
istioctl version: 1.8.4
Client Version: v1.19.4
Server Version: v1.18.10
version.BuildInfo{Version:"v3.3.4", GitCommit:"a61ce5633af99708171414353ed49547cf05013d", GitTreeState:"clean", GoVersion:"go1.14.9"}

**How was Istio installed?**
istiOperator

**Environment where the bug was observed (cloud vendor, OS, etc)**
AKS, linux
Additionally, please consider running `istioctl bug-report` and attach the generated cluster-state tarball to this issue.
Refer [cluster state archive](http://istio.io/help/bugs/#generating-a-cluster-state-archive) for more details.

**Comment by bianpengyuan** [comment]
istio-policy is part of Mixer which has already been remove at Istio 1.8. See https://istio.io/latest/news/releases/1.8.x/announcing-1.8/#deprecations thanks!

**Comment by shahriak** [comment]
Then is it being referred in other charts? I am not referring to istio-policy or istio-telemetry charts anywhere..

**Comment by bianpengyuan** [comment]
Oh I see. sorry, missed you question. The output is very hard to read. could you provide a well formatted values?

**Comment by shahriak** [comment]
Updated the format above.

**Comment by bianpengyuan** [comment]
Could you paste you operator CR?

**Comment by shahriak** [comment]
```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  namespace: istio-system
  name: istiocontrolplane
spec:
  profile: "default"
  hub: {{.Values.hub}}
  tag: {{.Values.tag}}
  installPackagePath: /istio/manifests

  meshConfig:
    defaultConfig:
      terminationDrainDuration: {{.Values.MeshConfig.TerminationDrainDuration}}

  components:
    ingressGateways:
    - name: istio-ingressgateway
      enabled: true
      k8s:
        env:
        - name: ISTIO_META_ROUTER_MODE
          value: "sni-dnat"
        - name: ISTIO_META_IDLE_TIMEOUT
          value: {{.Values.Gateway.IstioMetaIdleTimeout}}
        service:
          ports: 
          {{- range $port := .Values.Gateway.Ports }}
          - 
          {{- range $key, $value := $port }}
            {{ $key }}: {{ $value }}
          {{- end }}
          {{- end }}
        hpaSpec:
          minReplicas: {{.Values.Gateway.MinIngressReplicas}}
          maxReplicas: {{.Values.Gateway.MaxIngressReplicas}}
          # NOTE: do not remove the parameters below. these are the defaults for this istio component. these were added because configmerge will 
          # turn hpaSpec into null if the optional parameters aren't set without it. if hpaSec becomes null, then it blows away all of these
          # values in the parent helm chart.
          scaleTargetRef:
            apiVersion: apps/v1
            kind: Deployment
            name: istio-ingressgateway
          metrics:
            - type: Resource
              resource:
                name: cpu
                targetAverageUtilization: 60
        resources:
          requests:
            cpu: {{.Values.Gateway.RequestsCpu}}
            memory: {{.Values.Gateway.RequestsMemory}}
          limits:
            cpu: {{.Values.Gateway.LimitsCpu}}
            memory: {{.Values.Gateway.LimitsMemory}}
        strategy:
          rollingUpdate:
            maxSurge: "100%"
            maxUnavailable: "25%"
        nodeSelector:
          {{- range $key, $value := .Values.Gateway.NodeSelector }}
            {{ $key }}: {{ $value }}
          {{- end }}
        overlays:
          - kind: Deployment
            name: istio-ingressgateway
            patches:
              - path: spec.template.spec.dnsConfig
                value:
                  options:
                    - name: single-request-reopen
    pilot:
      enabled: true
      k8s:
        hpaSpec:
          minReplicas: 2
        overlays:
          - kind: Deployment
            name: istiod
            patches:
              - path: spec.template.spec.dnsConfig
                value:
                  options:
                    - name: single-request-reopen

    cni:
      enabled: {{.Values.Cni.Enabled}}
  values:
    global:
      trustDomain: {{.Values.clusterName}}
      multiCluster:
        enabled: true
        clusterName: {{.Values.clusterName}}
        globalDomainSuffix: "global"
        includeEnvoyFilter: true
      defaultNodeSelector:
        kubernetes.io/os: linux
      proxy:
        resources:
          requests:
            cpu: {{.Values.Global.Proxy.RequestsCpu}}
            memory: {{.Values.Global.Proxy.RequestsMemory}}
          limits:
            cpu: {{.Values.Global.Proxy.LimitsCpu}}
            memory: {{.Values.Global.Proxy.LimitsMemory}}
        holdApplicationUntilProxyStarts: true

    pilot:
      env:
        # pilot is replaced by istiod
        # Istio 1.3 added trust domain validation by default, but this breaks multi-trust domain scenarios, so we disable it
        PILOT_SKIP_VALIDATE_TRUST_DOMAIN: true

    # security:   # security field is deprecated in 1.6
      # By default Istio creates a self signed root certificate authority. If you are creating a multi-cluster mesh
      # (see Section 6), you should set this to false. This will tell Istio to look for your custom Root CA
      # certificate. If your mesh is single-cluster, feel free to omit this flag or set it to true.
      # selfSigned: false

    telemetry:
      v2:
        # if using mixer, we don't need new telemetry data flows
        enabled: false
    gateways:
      istio-ingressgateway:
        # preserve original client source IP, https://istio.io/latest/docs/tasks/security/authorization/authz-ingress/
        externalTrafficPolicy: Local
        ports:
        # We could enable this port if mtls is enabled on the all exposed ports but upstream load balancer requires an endpoint for health check
        # Istio/Envoy provides this port for default health check
        # - port: 15020
        #   targetPort: 15020
        #   name: status-port
        - port: 443
          targetPort: 32443
          name: https
        # This is to create 2 ingress gateway pods at minium if not speficied in "PlatformConfig.Deployment.IstioChartValues.Gateway.MinIngressReplicas" 
        # According to https://github.com/istio/operator, this is a pass-through API to helm values that we should avoid using in future. Validated "autoscaleMin" would be overwritten by hpaSpec -> minReplicas
        autoscaleMin: 2
        podAntiAffinityTermLabelSelector:
        - key: app
          operator: In
          values: istio-ingressgateway
          topologyKey: "kubernetes.io/os"
    cni:
      excludeNamespaces:
       - istio-system
       - kube-system
      logLevel: info
```

**Comment by bianpengyuan** [comment]
Ok thanks! I cannot find any issue in the operator config. Need environment maintainers to take a look. @morvencao @shamsher31 @richardwxn any idea?

**Comment by shahriak** [comment]
Closing! issue resolved after using 1.8.4 operator image
